### ISSUE TITLE
airodump-ng does not compile on Freebsd-7.1

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#606]
**Reported by anonymous on 31 Mar 2009 03:34 UTC**

```
%uname -a
FreeBSD dell.home 7.1-STABLE FreeBSD 7.1-STABLE #0: Mon Jan  5 14:10:55 CST 2009     milen@dell.home:/usr/obj/usr/src/sys/GENERIC  i386
%
%
%svn update
At revision 1498.
%
%date
Mon Mar 30 22:30:04 CDT 2009
%
%gmake -v
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-portbld-freebsd6.2
%
%gmake
gmake -C src all
gmake[Entering directory `/usr/home/milen/aircrack/aircrack-ng/src'
gmake -C osdep
gmake[2](1]:): Entering directory `/usr/home/milen/aircrack/aircrack-ng/src/osdep'
Building for FreeBSD
gmake[Entering directory `/usr/home/milen/aircrack/aircrack-ng/src/osdep'
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -fPIC -I..    -c -o osdep.o osdep.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -fPIC -I..    -c -o network.o network.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -fPIC -I..    -c -o freebsd.o freebsd.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -fPIC -I..    -c -o freebsd_tap.o freebsd_tap.c
ar cru libosdep.a  osdep.o network.o freebsd.o freebsd_tap.o
ranlib libosdep.a 
touch .os.FreeBSD
gmake[3](3]:): Leaving directory `/usr/home/milen/aircrack/aircrack-ng/src/osdep'
gmake[Leaving directory `/usr/home/milen/aircrack/aircrack-ng/src/osdep'
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o aircrack-ng.o aircrack-ng.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o crypto.o crypto.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o common.o common.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o uniqueiv.o uniqueiv.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o aircrack-ptw-lib.o aircrack-ptw-lib.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude aircrack-ng.o crypto.o common.o uniqueiv.o aircrack-ptw-lib.o sha1-sse2.S -o aircrack-ng -lpthread -lssl -lcrypto  
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o airdecap-ng.o airdecap-ng.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude airdecap-ng.o crypto.o common.o -o airdecap-ng -lssl -lcrypto 
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o packetforge-ng.o packetforge-ng.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude packetforge-ng.o common.o crypto.o -o packetforge-ng -lssl -lcrypto 
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o ivstools.o ivstools.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude ivstools.o common.o crypto.o uniqueiv.o -o ivstools -lssl -lcrypto 
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o kstats.o kstats.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude kstats.o -o kstats
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o makeivs-ng.o makeivs-ng.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude makeivs-ng.o common.o uniqueiv.o -o makeivs-ng
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o airdecloak-ng.o airdecloak-ng.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude airdecloak-ng.o common.o -o airdecloak-ng
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o aireplay-ng.o aireplay-ng.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude aireplay-ng.o common.o crypto.o -o aireplay-ng -Losdep -losdep   -lssl -lcrypto 
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude   -c -o airodump-ng.o airodump-ng.c
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1498  -Iinclude airodump-ng.o common.o crypto.o uniqueiv.o -o airodump-ng -Losdep -losdep   -lssl -lcrypto 
airodump-ng.o(.text+0x4286): In function `dump_write_kismet_netxml':
/usr/home/milen/aircrack/aircrack-ng/src/airodump-ng.c:3063: undefined reference to `getFrequencyFromChannel'
gmake[1](2]:): *** [Error 1
gmake[1](airodump-ng]): Leaving directory `/usr/home/milen/aircrack/aircrack-ng/src'
gmake: *** [all] Error 2
%
```

**Comment by aircrack-ng** [comment]
**Modified by misterx on 31 Mar 2009 03:34 UTC**

**Comment by aircrack-ng** [comment]
**Comment by misterx on 31 Mar 2009 03:34 UTC**

Fixed in aircrack-ng/aircrack-ng@00a9876ae1615c2f3c68a610e22368d8a6f6dc63
